src/core/navigation/Navigation.module.scss

```diff
@@ -0,0 +1,15 @@
+@import 'breakpoints';
+@import 'spacing';
+
+// WORKAROUND: Trying to set var(--header-max-width) to var(--breakpoint-xl) instead of
+//             the default 1440px. Not an ideal solution, this may break e.g. if HDS
+//             Header's subcomponents change their class names.
+.maxWidthXl {
+  div[class^='HeaderActionBar-module_headerActionBar_'],
+  div[class*=' HeaderActionBar-module_headerActionBar_'],
+  nav[class^='HeaderNavigationMenu-module_headerNavigationMenu_'],
+  nav[class*=' HeaderNavigationMenu-module_headerNavigationMenu_'] {
+    margin: 0 auto;
+    max-width: var(--breakpoint-xl);
+  }
+}
```

Comment:
@melniiv what do you think about this?

Reply:
i think that when HDS makes it configurable we refactor that.  Discussed with Kari

Reply:
Yes, this was discussed with HDS team on Slack #design-system [thread](https://helsinkicity.slack.com/archives/CHCV3KTHA/p1699606529886999) on Friday Nov 10th, 2023. It seemed HDS team was at least initially open to the idea of making the maximum header width configurable, but let's see!